.github/workflows/gh-pages.yaml

```diff
@@ -11,24 +11,52 @@ on:
     paths:
       - 'book/**'
 
+env:
+  CARGO_TERM_COLOR: always
+  # Update the language picker in index.hbs to link new languages.
+  LANGUAGES: zh
+
 jobs:
   deploy:
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-latest
+    environment:
+      name: github-pages
+      url: ${{ steps.deployment.outputs.page_url }}
     concurrency:
       group: ${{ github.workflow }}-${{ github.ref }}
+    defaults:
+      run:
+        shell: bash
     steps:
-      - uses: actions/checkout@v2
+      - name: Checkout
+        uses: actions/checkout@v3
 
-      # ref https://github.com/peaceiris/actions-mdbook
-      - name: Setup mdBook
-        uses: peaceiris/actions-mdbook@v1
-        with:
-          mdbook-version: 'latest'
-      
-      - name: Setup `mdbook-mermaid`
-        run: cargo install mdbook-mermaid
+      - name: Setup Rust cache
+        uses: Swatinem/rust-cache@v2
+
+      - name: Install mdbook
+        run: cargo install mdbook --locked --version 0.4.28
+
+      - name: Install mdbook-mermaid
+        run: cargo install mdbook-mermaid --locked --version 0.12.6
+
+      - name: Install mdbook-i18n-helpers
+        run: cargo install mdbook-i18n-helpers --locked --version 0.1.0
+
+      - name: Build book in English
+        working-directory: ./book
+        run: mdbook build -d book
 
-      - run: mdbook build ./book
+      - name: Build all translations
+        working-directory: ./book
+        run: |
+          for po_lang in ${{ env.LANGUAGES }}; do
+              echo "::group::Building $po_lang translation"
+              MDBOOK_BOOK__LANGUAGE=$po_lang \
+              MDBOOK_OUTPUT__HTML__SITE_URL=/flutter-rust-bridge/$po_lang/ \
+              mdbook build -d book/$po_lang
+              echo "::endgroup::"
+          done
```

Comment:
nit: it seems good to move them to `justfile`. Then, everyone who wants to play with it locally (including you and me ;) ) can execute a one-line command like `just whatever-command-name` instead of copy-and-paste these long code

Reply:
> what about adding this information at the top of each zh page

Not bad, hat about the people who contribute to the same page afterward?
But in any case, with the current translation process, it seems not so easy to add additional information to the translated page

Reply:
> it seems good to move them to justfile

agree, one thing to note is that if you want to serve all languages, you will have to first build and then use a separate http server to serve them, such as https://github.com/TheWaWaR/simple-http-server

Reply:
> Not bad, what about the people who contribute to the same page afterward?

Good question. Maybe just list all and add a link pointing to a separate page showing everyone's detailed contribution?

> But in any case, with the current translation process, it seems not so easy to add additional information to the translated page

Just a wild guess, Is it possible to modify the HTML?

> agree, one thing to note is that if you want to serve all languages, you will have to first build and then use a separate http server to serve them, such as [TheWaWaR/simple-http-server](https://github.com/TheWaWaR/simple-http-server)

I see. Then it would be great to have a bit of doc, telling users how to build and serve the doc locally